lib/es5-helpers.js

```diff
@@ -9,35 +9,27 @@ let prefixPath = null
 const parentPath = FS.realpathSync(Path.join(__dirname, '..'))
 const atomEslintPath = Path.join(parentPath, 'node_modules', 'eslint')
 
+// Find directory of eslint installation. Traverse up the tree checking every
+// `node_modules` directory
 function findEslintDir(params) {
-  const modulesPath = find(params.fileDir, 'node_modules')
-  let eslintNewPath = null
+  if (!params.global) {
+    return find(params.fileDir, ['node_modules', 'eslint'].join(Path.sep)) || atomEslintPath
```

Comment:
Looks like it does -> https://github.com/AtomLinter/atom-linter/blob/master/src/helpers.js#L210. Since it splits by Path.sep - rewrote that from `node_modules/eslint` to join(Path.sep) so it's always split correctly into chunks.